libpod/container_log.go

```diff
@@ -72,6 +72,12 @@ func (c *Container) readFromLogFile(ctx context.Context, options *logs.LogOption
 			}
 		}
 	}
+	go func() {
+		time.Sleep(time.Until(options.Until))
```

Comment:
If options.Until ==NIL, does this sleep forever?

BTW The commit message here is badly spelled.

Reply:
good catch, it also needs to ignore the case where until is the past